common/static/coffee/src/discussion/utils.coffee

```diff
@@ -1,16 +1,3 @@
-$ ->
-  if !window.$$contents
```

Comment:
Reviewers: note that I removed this ``$$contents`` variable after consultation with @jimabramson. It was a legacy piece of code that hasn't been used in a while. The remainder of this clause was converted into methods directly on ``DiscussionUtil`` below.